pyugrid/ugrid.py

```diff
@@ -38,6 +37,9 @@ class UGrid(object):
     The internal structure mirrors the netcdf data standard.
     """
 
+    # When reading from netCDF, check for Fortran v. C ordering.
+    check_array_order = True
```

Comment:
I not proud of this, but I haven't written any pyugrid code.   

I do remember making the argument that we could just check which dimension of the connectivity array is larger, and then flip it if necessary.   The argument being that this would work for any *real* case, since the most number of nodes in an element I can imagine is 8, and any real mesh will have more than 8 elements. 

But I agree that there should be a way to write a valid ugrid with 2 triangular elements.   :sad:

Instead of requiring the dimensions to be in a certain order, should we add an attribute to the connectivity array like `axes="node element"` or something?